web/react/stores/user_store.jsx

```diff
@@ -205,74 +255,76 @@ var UserStore = assign({}, EventEmitter.prototype, {
+    }
+    getStatuses() {
         return BrowserStore.getItem('statuses', {});
-    },
-    getStatus: function(id) {
+    }
+    getStatus(id) {
         return this.getStatuses()[id];
     }
-});
+}
+
+var UserStore = new UserStoreClass();
+UserStore.setMaxListeners(0);
 
-UserStore.dispatchToken = AppDispatcher.register(function(payload) {
+UserStore.dispatchToken = AppDispatcher.register(function registry(payload) {
     var action = payload.action;
 
     switch (action.type) {
-        case ActionTypes.RECIEVED_PROFILES:
-            for (var id in action.profiles) {
-                // profiles can have incomplete data, so don't overwrite current user
-                if (id === UserStore.getCurrentId()) {
-                    continue;
-                }
-                var profile = action.profiles[id];
-                UserStore.saveProfile(profile);
-                UserStore.emitChange(profile.id);
+    case ActionTypes.RECIEVED_PROFILES:
+        for (var id in action.profiles) {
+            // profiles can have incomplete data, so don't overwrite current user
+            if (id === UserStore.getCurrentId()) {
+                continue;
             }
-            break;
-        case ActionTypes.RECIEVED_ME:
-            UserStore.setCurrentUser(action.me);
-            UserStore.emitChange(action.me.id);
-            break;
-        case ActionTypes.RECIEVED_SESSIONS:
-            UserStore.setSessions(action.sessions);
-            UserStore.emitSessionsChange();
-            break;
-        case ActionTypes.RECIEVED_AUDITS:
-            UserStore.setAudits(action.audits);
-            UserStore.emitAuditsChange();
-            break;
-        case ActionTypes.RECIEVED_TEAMS:
-            UserStore.setTeams(action.teams);
-            UserStore.emitTeamsChange();
-            break;
-        case ActionTypes.RECIEVED_STATUSES:
-            UserStore._setStatuses(action.statuses);
-            UserStore.emitStatusesChange();
-            break;
+            var profile = action.profiles[id];
+            UserStore.saveProfile(profile);
+            UserStore.emitChange(profile.id);
+        }
+        break;
+    case ActionTypes.RECIEVED_ME:
+        UserStore.setCurrentUser(action.me);
+        UserStore.emitChange(action.me.id);
+        break;
+    case ActionTypes.RECIEVED_SESSIONS:
+        UserStore.setSessions(action.sessions);
+        UserStore.emitSessionsChange();
+        break;
+    case ActionTypes.RECIEVED_AUDITS:
+        UserStore.setAudits(action.audits);
+        UserStore.emitAuditsChange();
+        break;
+    case ActionTypes.RECIEVED_TEAMS:
+        UserStore.setTeams(action.teams);
+        UserStore.emitTeamsChange();
+        break;
+    case ActionTypes.RECIEVED_STATUSES:
+        UserStore.pSetStatuses(action.statuses);
+        UserStore.emitStatusesChange();
+        break;
 
-        default:
+    default:
     }
 });
 
-UserStore.setMaxListeners(0);
 global.window.UserStore = UserStore;
```

Comment:
Same as with the ConfigStore, I don't think we should be making this global when we don't need to be.